crates/server/src/authority/catalog.rs

```diff
@@ -418,10 +436,17 @@ async fn lookup<R: ResponseHandler + Unpin>(
         );
 
         let result = send_response(response_edns.clone(), response, response_handle.clone()).await;
-        if let Err(e) = result {
-            error!("error sending response: {}", e);
+
+        match result {
```

Comment:
Hoist the assignment out of the match?

Reply:
ended up removing this logic, multiple queries aren't technically supported anyway.